win32/tests/threadpool_win32_ut/threadpool_win32_ut.c

```diff
@@ -300,16 +297,29 @@ TEST_FUNCTION(threadpool_create_with_NULL_execution_engine_fails)
 }
 
 /* Tests_SRS_THREADPOOL_WIN32_01_001: [ threadpool_create shall allocate a new threadpool object and on success shall return a non-NULL handle. ]*/
-/* Tests_SRS_THREADPOOL_WIN32_42_027: [ threadpool_create shall increment the reference count on the execution_engine. ]*/
 /* Tests_SRS_THREADPOOL_WIN32_01_025: [ threadpool_create shall obtain the PTP_POOL from the execution engine by calling execution_engine_win32_get_threadpool. ]*/
+/* Tests_SRS_THREADPOOL_WIN32_01_026: [ threadpool_open shall initialize a thread pool environment by calling InitializeThreadpoolEnvironment. ]*/

```

Comment:
>threadpool_open [](http://example.com/codeflow?start=40&length=15)

In the test also: threadpool_open is does not exist any longer for all practical purposes